concepticondata/conceptlists/Kornai-2018-1400.tsv

```diff
@@ -1151,13 +1151,13 @@ Kornai-2018-1400-1149	1149	1640	spoil	U			megromlik	corrumpor	zepsuć sié	aft
 Kornai-2018-1400-1150	1150	1222	spoon	N	1378	SPOON	kanál	cochlear	łyźka	cutlery, [EAT soup] INSTRUMENT, HAS bowl, HAS handle
 Kornai-2018-1400-1151	1151	2124	sport	N			sport	gymnastica	sport	physical(activity), =AGT HAS rule, compete
 Kornai-2018-1400-1152	1152	2357	spread	V	38	SPREAD OUT	terít	sterno	szerzyć	
-Kornai-2018-1400-1153	1153	2358	spread	U			terjed	distendo	rozprzestrzeniać sié	after(=PAT AT/2744 wide)
+Kornai-2018-1400-1153	1153	2358	spread	U	38	SPREAD OUT	terjed	distendo	rozprzestrzeniać sié	after(=PAT AT/2744 wide)
 Kornai-2018-1400-1154	1154	1759	square	N	850	SQUARE	négyzet	quadratum	kwadrat	IN/2758 plane/2090, figure/140, HAS side[four, equal]
 Kornai-2018-1400-1155	1155	74	stand	U	1442	STAND	áll	sto	stać	=AGT[upright], =AGT ON two(foot)
 Kornai-2018-1400-1156	1156	1705	standard	N			minta	exemplum	przykład	[compare INSTRUMENT] CAUSE accept, society/2285 HAS, <moral>
 Kornai-2018-1400-1157	1157	2150	standard	A			szabványos	prototypus	standartowy	
 Kornai-2018-1400-1158	1158	408	star	N	1430	STAR	csillag	stella	gwiazda	planet, shine, dot, AT/2744 sky, AT/2744 night
-Kornai-2018-1400-1159	1159	1313	start	N			kezdet	initium	poczátek	after(=POSS)
+Kornai-2018-1400-1159	1159	1313	start	N	3447	BEGIN OR START	kezdet	initium	poczátek	after(=POSS)
```

Comment:
Unmap. BEGIN OR START is supposed to be the verb meaning.

Reply:
kezdet is a noun with the -et nominal derivational suffix, the verb is "kezd"